mimic_2_data_analysis.ipynb

```diff
@@ -45,23 +45,7 @@
    "execution_count": 1,
```

Comment:
<p>"Survival analysis is distinguished from other forms of analyses covered in this Chapter,"</p><p>Well, we are not writing a book here, but only a tutorial :)</p><p>Please try to copy a little bit less and to rephrase some things.</p><p> </p><p>Besides that, good!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='134'/>